package/android/src/main/java/com/shopify/reactnative/skia/SkiaDomView.java

```diff
@@ -14,6 +14,7 @@ public SkiaDomView(Context context) {
         super(context);
         RNSkiaModule skiaModule = ((ReactContext) context).getNativeModule(RNSkiaModule.class);
         mHybridData = initHybrid(skiaModule.getSkiaManager());
+        this.createSurfaceTexture();
```

Comment:
I don't know the codebase that well to tell, but what would be the reason for adding it in all the subclasses given it uses method from superclass. Shouldn't it be placed in the superclass. Also, my other suggestion to use onAttach event should eliminate a need to calling this in constructor altogether.